packages/reputation-miner/viz/setup.js

```diff
@@ -0,0 +1,99 @@
+/* globals artifacts */
+const { WAD, MINING_CYCLE_DURATION } = require("../../../helpers/constants");
```

Comment:
The setup here is in fact only hydrating some test data in the current cycle. I don't think it's necessary except for testing. In practice the visualiser should probably be a part of the `ReputationMiningClient` and started in its `constructor` or `initialise` call.

Reply:
You're right, it's only for testing. I can move it elsewhere to make that more clear.

Reply:
Maybe it fits in the `test-data-generator.js`? Or we don't need it at all? For development purposes we can also hook into the now live Goerli deployment where there are already some test log entries to process.

Reply:
I'd say leave it for now, since having more precise control over the test data will be useful for at least a while longer. I'd say leave it in `scripts` since truffle scripts run in a different environment than the test code.